cmd/daserver/daserver.go

```diff
@@ -126,20 +124,14 @@ func startup() error {
 	// Some different defaults to DAS config in a node.
 	das.DefaultDataAvailabilityConfig.Enable = true
 
-	vcsRevision, vcsTime := genericconf.GetVersion()
 	serverConfig, err := parseDAServer(os.Args[1:])
 	if err != nil {
-		fmt.Printf("\nrevision: %v, vcs.time: %v\n", vcsRevision, vcsTime)
-		printSampleUsage()
-		if !strings.Contains(err.Error(), "help requested") {
-			fmt.Printf("%s\n", err.Error())
-		}
+		util.HandleError(err, printSampleUsage)
 		return nil
 	}
 	if !(serverConfig.EnableRPC || serverConfig.EnableREST) {
-		fmt.Printf("\nrevision: %v, vcs.time: %v\n", vcsRevision, vcsTime)
+		printSampleUsage(os.Args[0])
 		fmt.Printf("Please specify at least one of --enable-rest or --enable-rpc\n")
```

Comment:
Maker cleaner to use `util.HandleError` and make `err` `errors.New("Please specify at least one of --enable-rest or --enable-rpc")`

Reply:
:+1: